### ISSUE TITLE
Build pyradio with `requests` and `dnspython`

### ISSUE THREAD

**Issue by j-steinbach** [NixOS/nixpkgs#151428]
###### Checklist

- [x] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name: pyradio

current version: 0.8.9
desired version: 0.8.9 built with `requests'; see [readme](https://github.com/coderholic/pyradio#packaging-pyradio)

###### Notify maintainers

maintainers: @contrun 

###### Note for maintainers

Please tag this issue in your PR.

----

I tried my hand on building this myself, but maybe someone experienced can take a look?

```nix
with import <nixpkgs> {};
#{ lib, python3Packages, fetchFromGitHub }:

python3Packages.buildPythonApplication rec {
  pname = "pyradio";
  version = "0.8.9.9";

  src = fetchFromGitHub {
    owner = "coderholic";
    repo = pname;
    rev = version;
    sha256 = "04asw5alkkf2q5iixswarj6ddb0y4a6ixm7cckl6204jiyxpv6kc";
  };

  propagatedBuildInputs = with python39Packages; [
    dnspython
    requests
  ];

  prePatch = ''
	  sed -i 's/distro = None/distro = NixOS/' pyradio/config
	  ''; 

  checkPhase = ''
    $out/bin/pyradio --help
  '';

  meta = with lib; {
    homepage = "http://www.coderholic.com/pyradio/";
    description = "Curses based internet radio player";
    license = licenses.mit;
    maintainers = with maintainers; [ contrun ];
  };
}
```

**Comment by j-steinbach** [comment]
Ah, the reason for this request is that the 'requests' module is required for the radio-browser.

![image](https://user-images.githubusercontent.com/69524139/146818383-70955f2c-2c35-419f-a2be-e7aab3513d5c.png)

**Comment by contrun** [comment]
This is great. Please submit a PR.

**Comment by j-steinbach** [comment]
@contrun I created a pull request (https://github.com/NixOS/nixpkgs/pull/151584). Can you take a look? I also added myself as a maintainer, based on the `We encourage people who care about a package to assign themselves as a maintainer.`

**Comment by tomodachi94** [comment]
Appears to have been fixed in #155600.
